### ISSUE TITLE
Failed to handle key=<Esc>. Cannot read property 'stop' of undefined

### ISSUE THREAD

**Issue by yuezk** [VSCodeVim/Vim#5736]
**To Reproduce**

1.  Go to '...'
2.  Click on '....'
3.  Scroll down to '....'
4.  See error

**VSCodeVim version**: 1.17.1

<details><summary>Stack trace</summary>

```
TypeError: Failed to handle key=<Esc>. Cannot read property 'stop' of undefined
    at p.get cursorStopPosition [as cursorStopPosition] (/Users/zyue/.vscode/extensions/vscodevim.vim-1.17.1/out/extension.js:9:505145)
    at F.runAction (/Users/zyue/.vscode/extensions/vscodevim.vim-1.17.1/out/extension.js:9:475433)
    at F.handleKeyAsAnAction (/Users/zyue/.vscode/extensions/vscodevim.vim-1.17.1/out/extension.js:9:474805)
    at F.handleKeyEvent (/Users/zyue/.vscode/extensions/vscodevim.vim-1.17.1/out/extension.js:9:471614)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at Object.promise (/Users/zyue/.vscode/extensions/vscodevim.vim-1.17.1/out/extension.js:9:231560)
    at Object.runTasks (/Users/zyue/.vscode/extensions/vscodevim.vim-1.17.1/out/extension.js:9:621914)
```

</details>

**Comment by edlucas** [comment]
I'm having the same issue this morning, for the first time.

**Comment by MCluck90** [comment]
Repro steps:

1. Open a new untitled file (or any other file)
2. Set the language mode to JavaScript, TypeScript, or any of the React variants
3. Insert the following input

```ts
'a'
```

4. Move caret to `a`

```ts
'a'
 ^
```

5. Type x to try and delete `a`

**Comment by dustinm76** [comment]
Disabling the Template String Converter extension fixed this for me. Even though the error is coming from the vim extension.

**Comment by joshmu** [comment]
Rather than disabling, rolling back 'Template String Converter' extension to 0.3.0 removes the error.

**Comment by meganrogge** [comment]
resolved in Template String Converter v 0.3.2. thanks for bringing this to my attention @joshmu @dustinm76

**Comment by jdugan1024** [comment]
FWIW -- I had to uninstall the extension and then do:

    rm -rf  ~/.vscode/extensions/vscodevim.vim-1.17.1

And then reinstall via the extensions marketplace. Seems a little excessive but I don't fully understand how all the pieces fit together.

Leaving this here in case someone else has a similar problem.
